libsolidity/analysis/OverrideChecker.h

```diff
@@ -84,7 +84,10 @@ class OverrideProxy
 	/// @returns receive / fallback / function (only the latter for modifiers and variables);
 	langutil::Token functionKind() const;
 
-	FunctionType const* functionType() const;
+	/// @returns the externally callable function type
+	FunctionType const* externalFunctionType() const;
+	/// @returns the (unmodified) function type
+	FunctionType const* specificFunctionType() const;
```

Comment:
Maybe "original" or "native" would be a better word here?
```suggestion
	/// @returns the function type exactly as in its original definition
	FunctionType const* originalFunctionType() const;
```